extensions/identity-hub-verifier/src/main/java/org/eclipse/dataspaceconnector/identityhub/verifier/IdentityHubCredentialsVerifier.java

```diff
@@ -138,4 +143,10 @@ private String getIdentityHubBaseUrl(DidDocument didDocument) {
                 .map(Service::getServiceEndpoint)
                 .orElse(null);
     }
+
+    private static class VerificationResult extends AbstractResult<Map<String, Object>, Failure> {
```

Comment:
The namings seem a bit off: the method is called `extractClaimsFromCredential` but the result is a `VerificationResult` which gets assigned to a variable called `claims`. What about renaming this class to `ClaimsExtractionResult`?

Reply:
Ok agreed, lets discuss in the next PR